homeassistant/components/panasonic_viera/device_trigger.py

```diff
@@ -0,0 +1,80 @@
+"""Provides device automations for control of Panasonic Viera."""
```

Comment:
This seems like a misleading docstring.
How can a device trigger *control* anything?